ibm/service/vpc/resource_ibm_is_backup_policy.go

```diff
@@ -127,6 +182,17 @@ func ResourceIBMIsBackupPolicyValidator() *validate.ResourceValidator {
 			MaxValueLength:             128,
 		},
 	)
+	validateSchema = append(validateSchema,
```

Comment:
We don't need a validator for TypeList schema argument scope. This can be removed.